app/views/users/show.html.erb

```diff
@@ -0,0 +1,13 @@
+<p id="notice"><%= notice %></p>
```

Comment:
A flash uzeneteket en lehet egy partialbe raknam, vagy egyenesen a application.html.erb-be hogy minden oldalon megjelenjen ha szukseges.